.travis.yml

```diff
@@ -13,6 +13,7 @@ script:
   - yarn run test:unit
   - yarn run test:e2e
   - yarn run build
+  - echo "config.qmk.fm" > dist/CNAME
```

Comment:
You should be able to use the deploy `fqdn` option <https://docs.travis-ci.com/user/deployment/pages/#further-configuration>